core/src/main/java/psiprobe/AbstractTomcatContainer.java

```diff
@@ -406,7 +406,7 @@ public void listContextJsps(Context context, Summary summary, boolean compile) {
   }
 
   @Override
-  public org.apache.juli.logging.Log getLogger(Context context) {
+  public Object getLogger(Context context) {
```

Comment:
Is it ok to do that on a separate Pull Request?  I don't see much harm letthing this one through then following up with that fix.